kubechain/internal/controller/taskruntoolcall/taskruntoolcall_controller.go

```diff
@@ -441,7 +441,7 @@ func (r *TaskRunToolCallReconciler) updateTRTCStatus(ctx context.Context, trtc *
 
 	// Store the result for tool call rejection
 	if trtcStatusPhase == kubechainv1alpha1.TaskRunToolCallPhaseToolCallRejected {
-		trtcDeepCopy.Status.Result = result
+		trtcDeepCopy.Status.Result = "Human rejected tool call with the following response: " + result
```

Comment:
can we make this match exactly what the SDK does right now? 

https://github.com/humanlayer/humanlayer/blob/main/humanlayer/core/approval.py#L270

https://github.com/humanlayer/humanlayer/blob/main/humanlayer/core/approval.py#L268

https://github.com/humanlayer/humanlayer/blob/main/humanlayer/core/approval.py#L205